style.css

```diff
@@ -0,0 +1,95 @@
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body{
+  margin: 0 auto;
+}
```

Comment:
- Move ` *{ ... } ` to a separate `reset.css` file
- the reset already has `margin:0`. it's not necessary within  `body`